stylesheets/components/button.css

```diff
@@ -0,0 +1,163 @@
+@import url("/stylesheets/main.css");
+
+.btn {
+  margin: 1rem;
+  padding: 0.5rem;
+  font-size: 15px;
+  min-width: 80px;
+  letter-spacing: 1px;
+  border-radius: 0.2rem;
+  border: 1px solid;
+  background-color: var(--bg-white);
+}
+
+.btn-dark {
+  background-color: black;
+  color: white;
+  border: 1px solid black;
+}
+
+.btn-lg {
```

Comment:
Don't give a fixed font size. You can use rems.

Reply:
Thanks , will keep that in mind.